ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -442,11 +462,58 @@
                     root: 'data',
                     fields: ['uid', 'name']
                 }),
+                listeners: {
+                    select: function (combo) {
+                        var reportID = combo.getValue(),
+                            graphGroupCombo = Ext.getCmp('graphGroupCombo'),
+                            store = graphGroupCombo.getStore();
+                        store.load({
+                            params: {
+                                uid: reportID
+                            }
+                        });
+                        graphGroupCombo.setDisabled(false);
+                    }
+                },
                 displayField: 'name',
                 valueField: 'uid',
                 fieldLabel: _t('Multi-Graph Report (created on Reports screen)'),
                 value: this.reportUid
-            }];
+            }, {
+                id: 'graphGroupCombo',
+                xtype: 'combo',
+                name: 'graphGroup',
+                queryMode: 'local',
+                store: groupStore,
+                displayField: 'id',
+                valueField: 'id',
+                fieldLabel: _t('Graph Group'),
+                disabled: !(this.graphGroup),
+                value: this.graphGroup
+            }, {
+                id: "rangePicker",
+                xtype:'combo',
+                fieldLabel: _t('Range'),
+                name: 'drange',
+                editable: false,
+                forceSelection: true,
+                autoSelect: true,
+                value: '1h-ago',
+                queryMode: 'local',
+                valueField: 'id',
+                displayField: 'name',
+                store: new Ext.data.ArrayStore({
+                    id: 0,
+                    model: 'Zenoss.model.IdName',
+                    data: DATE_RANGES
+                }),
+                listeners: {
+                    select: function (combo) {
+                        var drange= combo.getValue()
+                    }
+                }
```

Comment:
Here's a note to check this listener, to see if it's needed on the date range picker.